app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -34,6 +39,7 @@
 import org.cru.godtools.shortcuts.GodToolsShortcutManager;
 import org.cru.godtools.shortcuts.GodToolsShortcutManager.PendingShortcut;
 import org.cru.godtools.util.ActivityUtilsKt;
+import org.jetbrains.annotations.NotNull;
```

Comment:
you should use the AndroidX `@NonNull` annotation instead of the jetbrains `@NotNull` annotation